csi/watch-logrotate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+while true
+do
+  if [[ $(ls /var/log/gluster/*.gz | wc -l) -ge 3 ]]; then
+    rm -rf /var/log/gluster/*.gz
+  fi
```

Comment:
Do we need this? logrotate takes care of deleting old files more than the rotate count specified in the conf?

Reply:
During the test it did not remove the .gz files even after a long time. So added this.